package.json

```diff
@@ -3,8 +3,9 @@
   "scripts": {
     "a11y": "metal-a11y --packages ./packages -c demos/index.html",
     "build": "lerna run build",
-    "ci": "npm run build && npm run checkFormat && npm run lint && npm run test",
+    "ci": "npm run build && npm run checkFormat && npm run lint && npm run test && npm run coveralls",
```

Comment:
What do you think the using CI configurations in `travis.yml`?

```yml
script:
  - npm run build
  - npm run checkFormat
  - npm run lint
  - npm run test

# Coveralls
after_success: npm run coveralls
```

Reply:
Sounds good!